journal/202308.md

```diff
@@ -0,0 +1,870 @@
+
+_Image: Cryptopower can [convert coins](https://twitter.com/exitusdcr/status/1699366344813109572) without touching the browser { confirm }_
+
+
+<a id="bison-relay-v018-release" />
+
+## Bison Relay v0.1.8 Release
+
+This is a big release with exciting new features and polish from 4 months of development.
+
+Major changes in both GUI and text apps:
+
+- Loading of chat history after app restarts
+- Pages with hyperlinks (similar to web pages)
+- Improved reliability of key exchanges and tipping
+- Numerous bug fixes
+
+GUI app highlights:
+
+- Address book with searching and filtering
+- News Feed can now show images and has improved sorting
+- Improved notifications UI
+- Improved responsiveness on smaller screens
+
+brclient text app highlights:
+
+- Initial e-commerce features
+- Client-side content filtering
+- Client data backups
+
+The initial e-commerce infrastructure called "simplestore" is designed for selling digital-only goods such as images, videos, audio, and files. Supported payment methods are DCR Lightning Network, on-chain DCR transfers, and custom. Simplestore is only available in the text-based brclient app in this release.
+
+GUI and text clients have different features now. Changes for each app are listed on the [release page](https://github.com/companyzero/bisonrelay/releases/tag/v0.1.8), and mind the upgrade notice if migrating from v0.1.7 on Windows. For a complete list of changes with explanations see [Decred Journal](https://xaur.github.io/decred-news/) issues from April to June 2023. See the [README](https://github.com/companyzero/bisonrelay/blob/v0.1.8/README.md#verifying-binaries) file for instructions on verifying the files.
+
+
+<a id="development" />
+
+## Development
+
+The work reported below has the "merged to master" status unless noted otherwise. It means that the work is completed, reviewed, and integrated into the source code that advanced users can [build and run](https://medium.com/@artikozel/the-decred-node-back-to-the-source-part-one-27d4576e7e1c), but is not yet available in release binaries for regular users.
+
+
+### dcrd
+
+_[dcrd](https://github.com/decred/dcrd) is a full node implementation that powers Decred's peer-to-peer network around the world._
+
+The majority of the work this month was developer-focused, and not necessarily user-focused. However, it's always interesting when a new Go version is supported, because it unlocks potential new features and improvements from the coding language itself (reference the dcrwallet updates from the [July issue](202307.md#dcrwallet) for examples of this).
+
+The following work has been merged in `master` towards future releases:
+
+- Dropped official support for Go 1.19, and [added support for Go 1.21](https://github.com/decred/dcrd/pull/3172) (Go 1.20 is still supported as well). This included updates to the GitHub Actions build workflow.
+  - { _let's clarify what "official support" is, i.e. is building with Go 1.19 still possible as go.mod says?_ }
```

Comment:
These are the versions that the software is extensively tested against.  You can still build against Go 1.19, and it is highly likely to work fine, but it is just no longer tested.